app/Entry.js

```diff
@@ -23,7 +23,7 @@ import Onboarding1 from './views/onboarding/Onboarding1';
 import Onboarding2 from './views/onboarding/Onboarding2';
 import Onboarding3 from './views/onboarding/Onboarding3';
 import Onboarding4 from './views/onboarding/Onboarding4';
-import Onboarding5 from './views/onboarding/Onboarding5';
+import OnboardingPermissions from './views/onboarding/OnboardingPermissions';
```

Comment:
Yes! renaming this is helpful. I have never remembered which OnboardingX is which.